### DISCUSSION TITLE
Webhooks without Payload

### DISCUSSION THREAD

**Discussion by iam-abbas** [strapi/strapi#7931] [Legacy Topics]
Is there any way to trigger webhooks without default payload?

When triggered the webhook it sends the following data as payload:
```
{
  "event": "entry.create",
  "created_at": "2020-01-10T08:47:36.649Z",
  "model": "address",
  "entry": {
    "id": 1,
    "geolocation": {},
    "city": "Paris",
    "postal_code": null,
    "category": null,
    "full_name": "Paris",
    "created_at": "2020-01-10T08:47:36.264Z",
    "updated_at": "2020-01-10T08:47:36.264Z",
    "cover": null,
    "images": []
  }
}
```

I am trying to trigger a CircleCI endpoint to build/deploy a Gatsby application.  When I use the CircleCI API Endpoint as the URL for Strapi Webhook I get the following response:
```
{ "message" : "Unexpected parameter(s): :event, :created_at" }
```

So is there any way to send an empty payload? I also tried to create my own webhook using [this guide](https://strapi.io/blog/strapi-webhooks) but it did not seem to work.

**Comment by lauriejim** [comment]
Hello! It's currently not possible to customize the data you are sending in webhooks.
You can customize the headers but in your case, it's not what you want.
About the blog post you linked, it's related to the previous version of Strapi and the life cycle functions changed a little bit.
The overall logic still works but here is the documentation on the life cycle - https://strapi.io/documentation/v3.x/concepts/models.html#lifecycle-hooks
This will help you to set up things properly.
